README.md

```diff
@@ -126,6 +126,17 @@ npm run bundle
 These commands utilize [esbuild](https://esbuild.github.io/getting-started/#bundling-for-node) to bundle the action and
 its dependencies into a single file located in the `dist` folder.
 
+## Incrementing the Version
+This action uses [git-version-lite] to examine commit messages to determine whether to perform a major, minor or patch increment on merge.  The following table provides the fragment that should be included in a commit message to active different increment strategies.
+| Increment Type | Commit Message Fragment                     |
+| -------------- | ------------------------------------------- |
+| major          | +semver:breaking                            |
+| major          | +semver:major                               |
+| minor          | +semver:feature                             |
+| minor          | +semver:minor                               |
+| patch          | *default increment type, no comment needed* |
```

Comment:
I really like this! It isn't super hard to go find git-version-lite and look at its README, but having this here is just that much easier. If it were something that could see a lot of change I wouldn't want to include it since we'd have to update git-version-lite's README and this. Since that isn't the case I love having this here!